### PR TITLE
CocoaPods Mac app blog post

### PR DESCRIPTION
**Pull request by orta**
- [x] intro / TLDR
- [x] solid center
- [x] wrap up

### CODE REVIEW

**Comment by orta** [comment]
OK this is now :+1: from me

File: _posts/2015-12-31-CocoaPods-App.markdown [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+
+The GUI aspect of the app is built mainly in Swift and Objective-C when necessary. It uses all the fancy modern goodies available in the latest Cocoa APIs: NSViewControllers, Storyboards and [XPC](https://www.objc.io/issues/14-mac/xpc/).
+
+{% breaking_image /assets/blog_img/app/storyboards.png, /assets/blog_img/storyboards.png, class="image-zoom" %}
+
+It's cleanly built using simple composition, and has been the source for quite a few people's first open source contribution. We keep a well stocked [issue cabinet](http://github.com/cocoapods/cocoapods-app/issues) with well marked out issues for people wanting to help out. I aim to always put up designs early for features, making it easy for people to contribute with a specification in mind.
+
+#### pod
+
+We doing plan  on replicating every feature of the command-line gem in the CocoaPods app. That's just not good usage of our time. Instead we offer a way to access the command-line interface with the version of CocoaPods that is inside CocoaPods.app. We do this in the same way Xcode does, by offering command line tools:
```
Comment (alloy) [inline _posts/2015-12-31-CocoaPods-App.markdown]: s/doing/don't/ ?

File: _posts/2015-12-31-CocoaPods-App.markdown [hunk 2]
```diff
@@ -0,0 +1,76 @@
+---
+layout: post
+title:  "App 1.0"
+author: orta
+categories: cocoapods app release
+post_title_id:  "app"
+---
+
+TLDR: CocoaPods now has a Mac App that provides a hosted ruby experience, go check out the web page for it: [cocoapods.org/app](http://cocoapods.org).
```
Comment (KrauseFx) [inline _posts/2015-12-31-CocoaPods-App.markdown]: `https`

Comment (KrauseFx) [inline _posts/2015-12-31-CocoaPods-App.markdown]: `ruby` upper case maybe?

File: _posts/2015-12-31-CocoaPods-App.markdown (hunk 1 above)
Comment (KrauseFx) [inline _posts/2015-12-31-CocoaPods-App.markdown]: @orta @alloy yeah

**Comment by AliSoftware** [comment]
Doesn't the CP app support Podfile syntax validation yet, or is it still WIP? If it's already available that's one key feature this post should talk about IMHO to engage people in using the app rather that the gem, right?

File: _posts/2015-12-31-CocoaPods-App.markdown [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+
+As it has the same name, you don't need to change any scripts or your development flow in the terminal once installed. I've been using it as my main `pod` command for the last few months.
+
+#### ReflectionService
+
+The CocoaPods app needs to talk to the `pod` command provided by ruby. The simplest option is via [NSTask](http://www.raywenderlich.com/36537/nstask-tutorial), which will execute a command in a shell and output text to `STDIN` and `STERR`. This makes it easy to run commands, but it can be hard to deal with interactivity. It's effectively a one way communication pattern.
+
+There have been a few tools which bridge Ruby and Cocoa, the most notable being [RubyMotion](http://www.rubymotion.com). There is however a bridge that comes shipped with every Mac OS. That is [RubyCocoa](http://rubycocoa.github.io). We built support for bi-directional communication between Objective-C and Ruby using RubyCocoa.
```
Comment (alloy) [inline _posts/2015-12-31-CocoaPods-App.markdown]: RC doesn’t come with OS X anymore.

**Comment by orta** [comment]
Thanks everyone, addressed all feedback

**Pull request merged**